telemetry/specs/INT.mdk

```diff
@@ -854,11 +859,14 @@ each bit corresponds to a specific standard metadata as specified in Section 3.
   - bit5: Egress timestamp
   - bit6: Level 2 Ingress Port ID + Egress Port ID (4 bytes each)
   - bit7: Egress port Tx utilization
+  - bit14: Domain Specfic Metadata
   - bit15: Checksum Complement
   - The remaining bits are reserved.
   Each instruction requests 4 bytes of metadata to be inserted at each hop,
-  except if bit 6 is set, which requires 8 bytes of metadata. Per-hop
-  metadata length is set accordingly at the INT source.
+  except if bit 6 and bit 14 is set. If bit 6 is set, the instruction requires 
+  8 bytes of metadata. If bit 14 is set, the instruction requires a domain specific
+  metadata of n bytes, n being a multiple of 4 bytes. Per-hop metadata length is 
+  set accordingly at the INT source.
```

Comment:
If only the source can set per-hop metadata length in accordance with the domain specific metadata, then a INT transit node should not be allowed to set the D bit as the per-hop metadata length will no longer be accurate.

Reply:
We discussed this bit in the WG meeting. I will add text to explain the use of this bit.